programs/degen-pools/src/pools.rs

```diff
@@ -19,6 +19,12 @@ pub struct PoolCreated {
     pub description: String,
 }
 
+#[event] 
+pub struct PoolPaused {
```

Comment:
`PoolPaused` is a slightly misleading name, no? This only becomes a pause event if `is_paused` below is true. I for one made the mistake of thinking this is emitted only when the pool is paused (when I was writing the [backend event handler](https://github.com/Degen-Markets/degen-markets-be/commit/e61f0279c49a03314e2eafb6efa1687ec9aa12b2#diff-6eed9306ffc2efcb2cd3e13c4cdaced86c6c7a4d59bef0a952ee3d31d0213322R17)) 🙈 

Is there some other name we can think of? `PauseStatus`? `PauseSet`? `PoolStatus`? Dammit, nothing is as elegant as `PoolPaused`. @AngSin any ideas for names?

Reply:
Good point, how about `PoolStatusUpdated` ? Or how about keeping `PoolPaused` but passing the boolean in the event?

Reply:
@AngSin Event already has the boolean. Just the event name felt misleading (like the boolean was irrelevant). Any name that doesn't overrule the boolean's meaning is great (your suggestion fits this too)

@QalbeHabib your call

Reply:
Thanks for the suggestions!
I agree that `PoolStatusUpdated ` works well to reflect both states without confusion.
Let's go with that if everyone is on board.